src/main/java/propertyFiles/Bundle.properties

```diff
@@ -292,6 +293,8 @@ notification.typeDescription.WORKFLOW_FAILURE=External workflow run has failed
 notification.typeDescription.STATUSUPDATED=Status of dataset has been updated
 notification.typeDescription.DATASETCREATED=Dataset was created by user
 notification.typeDescription.DATASETMENTIONED=Dataset was referenced in remote system
+notification.typeDescription.PIDRECONCILED=The Persistent identifier of dataset has been updated
+
```

Comment:
is a notification.email.pidreconciled needed as well (somewhere around line 800)? I think sendNotification does try to send email unless this notification is muted.

Reply:
Thanks for hint. I wasn't aware the system notifications and email are handled differently. I've added the needed adaptions. Shall I add a testcase for the email construction or is the attached screenshot OK?
![Screenshot 2024-08-05 at 14 53 08](https://github.com/user-attachments/assets/2654aa3a-3069-46ef-818b-bea05c8594d0)